src/data/hooks/useWithdrawRequestStatus.ts

```
@@ -0,0 +1,95 @@
+  useEffect(() => {
+    const checkWithdrawRequest = async () => {
+      if (chainId !== cellarConfig.chain.wagmiId) return
+        try {
+          if (
+            walletClient &&
+            withdrawQueueContract &&
+            address &&
+            cellarConfig &&
+            !boringQueueWithdrawals
+          ) {
+            // @ts-ignore
```

ESLint

[error] Do not use "@ts-ignore" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)